### PR TITLE
General fixed 145

### PR DESCRIPTION
**Pull request by steichert**
General fixes to the front end.

### CODE REVIEW

File: frontend/app/bot-config/team-desires/team-desires.component.ts [hunk 1]
```diff
@@ -127,6 +134,18 @@ export class TeamDesiresComponent implements OnInit {
         }
         compoundCondition.logicalOperators[index] = op;
     }
+
+    togglePanel() {
```
Comment (vector67) [inline frontend/app/bot-config/team-desires/team-desires.component.ts]: Why did you add this function?

Comment (steichert) [reply frontend/app/bot-config/team-desires/team-desires.component.ts]: Its used to add and remove an active class on a Team Desires panel, which adds the necessary css to animate the arrow when clicked.

Comment (vector67) [reply frontend/app/bot-config/team-desires/team-desires.component.ts]: Cool, We need to work out a better way of doing this, but for now it's fine.
